parquet-avro/src/test/java/org/apache/parquet/avro/TestReadWrite.java

```diff
@@ -1,14 +1,14 @@
-/* 
+/*
```

Comment:
(nit) Unrelated formatting-only change.

Reply:
This is done by my IDE. Personally, I don't like trailing whitespace, and since it doesn't affect any code, I would just remove it. We don't lose any important lineage which is the reason to not touch unrelated stuff.